test/integration/framework/perf_utils.go

```diff
@@ -109,13 +110,13 @@ func (p *IntegrationTestNodePreparer) PrepareNodes(ctx context.Context, nextNode
 			}
 		}
 		if err != nil {
-			klog.Fatalf("Error creating node: %v", err)
+			return fmt.Errorf("creating node: %w", err)
 		}
 	}
 
 	nodes, err := waitListAllNodes(p.client)
 	if err != nil {
-		klog.Fatalf("Error listing nodes: %v", err)
+		return fmt.Errorf("listing nodes: %w", err)
```

Comment:
Could you also remove the logging from the line 125? I think we don't need to double log `Aborting node preparation`.

Reply:
Done. I'm wrapping the error to preserve the bread crumbs when that path is taken.